docs/docs/dcap.md

```diff
@@ -1,96 +1,70 @@
-    and any future generations. Please ensure you are using DCAP attestation on these platforms.
+## DCAP Client
 
+To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is
+hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to
+date. A DCAP client is used for this function. It is basically a library named `libdcap_quoteprov.so.1` with a 
+symbolic link at `libdcap_quoteprov.so`, which is to be installed in `/usr/lib/` or `/usr/lib/x86_64-linux-gnu`.
 
-In DCAP, repeated attestation requests aren't forwarded to Intel, but rather served from a cache. A newly installed
-machine obtains a machine certificate from Intel via the cache, which may then be persisted to disk. All this is
-automated for you.
+You can use one of the three available DCAP clients listed below:
 
-As cloud providers run caches, DCAP supports vendor-specific plugins. Intel provides a default one
-which requires a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification).
+1. Intel DCAP client
+2. Azure DCAP client
+3. Conclave-Azure bundled client
 
-Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. Conclave
-bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
-comes with the necessary libraries bundled, you don't need to do any further setup.
+To avoid conflicts between DCAP client plugins, you must uninstall the plugins that you don't require.
 
-### DCAP Plugin
-To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to date.
+### Intel DCAP Client
 
-DCAP is designed to work on many different server topologies. So, rather than directly connecting to Intel services to retrieve this information, the cloud vendor or the owner of the SGX system must provide a DCAP client plugin with the required information. Intel provides a generic DCAP client plugin as part of the DCAP runtime. To use this, you also need to set up a Provisioning Certificate Caching Service (PCCS). Intel provides an example and some instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). If you would like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs), then additional work may be required to provide data in the correct format for Conclave. Please contact R3 support if you need to set this up.
+Intel provides a DCAP client plugin as part of the DCAP runtime.
```

Comment:
Here I would make it clear why the users might want to use, i.e. then they need to build your own "DCAP infrustructure" and don't want to rely on cloud providers.
I would clarify like this a bit
```
Intel provides a DCAP client plugin as part of the DCAP runtime; you generally use this when you don't want to use cloud providers and you want to run your code on bare-metal machines.
To do that, you need to install the Intel DCAP client, for example in Ubuntu you can run this...

```